account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -423,12 +435,31 @@ def create_invoice(self):
         invoice = self._create_invoice(parsed_inv)
         invoice.message_post(_(
             "This invoice has been created automatically via file import"))
+        currency = self.env['res.currency'].search([
+            ('name', '=', parsed_inv.get('currency'))
+        ])
+        if not currency:
+            # Currency not found, fallback to company
+            currency = self.env.user.company_id.currency_id
+
+        if (currency.compare_amounts(
+                invoice.amount_total, parsed_inv.get('amount_total')) != 1):
```

Comment:
Surprised by the `!= 1`
```
>>> self.env.user.company_id.currency_id.compare_amounts(1.0, 1.0)
0
```

Reply:
BTW you already have an invoice record at this point, so please take this invoice's currency_id.

Reply:
I'm suprised to what happend with my test :(  Tested again, and should indeed be `!= 0`